spec/controllers/media_controller_spec.rb

```diff
@@ -204,6 +204,29 @@
 
         expect(response.headers['Cache-Control']).to eq('max-age=86400, public')
       end
+
+      context 'and the replacement is draft' do
+        before do
+          replacement.update_attribute(:draft, true)
+        end
+
+        it 'serves the asset when requested via something other than the draft-assets host' do
```

Comment:
Minor: "serves the asset" -> "serves the original asset"; or "serves the asset" -> "serves the replaced asset"